deployment/stacks/env-config.template.js

```diff
@@ -0,0 +1,17 @@
+window._env_ = {
+  API_REGION: "us-east-1",
```

Comment:
This looks never to be used anywhere. Going to remove from seds also because of this.